packages/application-shell/src/components/login/countdown.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import InfoDialog from '../../from-core/info-dialog';
+import styles from './login.mod.css';
+import messages from './messages';
+
+const initialCount = 3;
+
+export class Contdown extends React.PureComponent {
```

Comment:
Count

Reply:
Yes thanks, it's fixed in the follow up PR 😉